charts/clamav/templates/clamav-notify-config.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "clamav.fullname" . }}-notify-config
+data:
+  freshclam.conf: |
+    Checks {{ .Values.clamavNotify.freshclam.checkFrequency }}
+    CompressLocalDatabase yes
+    DatabaseOwner clam
+    {{- range .Values.clamavNotify.freshclam.mirrors }}
+    DatabaseMirror {{ . }}
+    {{- end }}
+    TestDatabases yes
+    LogVerbose yes
+    NotifyClamd no
+  lighttpd.conf: |
+    server.document-root = "/var/lib/clamav/mirror/"
+
+    server.port = 8080
+
+    mimetype.assign = (
+      ".htm" => "text/html", 
```

To be consistent with the other suggestions
```suggestion
      ".html" => "text/html", 
```

---

As above